### PR TITLE
Skip irrelevant patches

### PR DESCRIPTION
**Pull request by BaurzhanSakhariev**
see commits

### CODE REVIEW

File: devs/docs/es-backports.rst [hunk 1]
```diff
@@ -82,45 +82,45 @@ should be crossed out as well.
 - [ ] 745f527feac Deduplicate Index Meta Generations when Deserializing (#65619) (#65666)
 - [x] 6bbeedc9321 Reset Deflater/Inflater after Use in DeflateCompressor (#65617) (#65646)
 - [ ] 0137c1679bb Fix the earliest last modified age of translog stats (#64753)
-- [ ] fb84b6710d5 Restore use of default search and search_quote analyzers (#65491) (#65562)
-- [ ] 88993e763f9 Fix handling of null values in geo_point (#65307)
+- [s] fb84b6710d5 Restore use of default search and search_quote analyzers (#65491) (#65562)
+- [s] 88993e763f9 Fix handling of null values in geo_point (#65307)
 - [ ] feca22729cd [DOCS] Remove duplicated word in replica shard allocator comment (#65295) (#65317)
-- [ ] caf143f4a51 Unused boost parameter should not throw mapping exception (#64999) (#65014)
+- [f] caf143f4a51 Unused boost parameter should not throw mapping exception (#64999) (#65014)
```
Comment (seut) [inline devs/docs/es-backports.rst:88]: shouldn't this be `s` or `su`?

**Review by seut — COMMENTED** [review]
You could use `su` in most cases I think, but `s` is also fine.
Found one typo?

File: devs/docs/es-backports.rst (hunk 1 above)
Comment (BaurzhanSakhariev) [reply devs/docs/es-backports.rst]: yes, thx for catching!

**Comment by BaurzhanSakhariev** [comment]
> You could use `su` in most cases I think

I updated all commits except FieldMapper skipping as we have such functionality/code just skipping it due to https://github.com/crate/crate/issues/10025

**Review by seut — APPROVED** [review]
👍

**Pull request merged**